shared/templates/create_sysctl.py

```diff
@@ -3,34 +3,29 @@
 from template_common import FilesGenerator, UnknownTargetError, CSVLineError
 
 
+ONE_FILE_OVAL_OUTPUT = False
+
+
 class SysctlGenerator(FilesGenerator):
     def is_ipv6_id(self, var_id):
         return var_id.find("ipv6") >= 0
 
-    def get_files_var_for_id(self, var_id):
+    def get_files_for_id(self, var_id):
 
         if self.is_ipv6_id(var_id):
-            template_name = 'template_OVAL_sysctl_ipv6'
+            template_type = 'I'  # I = ipv6
         else:
-            template_name = 'template_OVAL_sysctl'
-
-        return {
-            'template_OVAL_sysctl_static_var': 'sysctl_static_',
-            'template_OVAL_sysctl_runtime_var': 'sysctl_runtime_',
-            template_name: 'sysctl_'
-        }
+            template_type = 'P'  # P = plain
 
-    def get_files_for_id(self, var_id):
-        if self.is_ipv6_id(var_id):
-            template_name = 'template_OVAL_sysctl_ipv6'
+        if ONE_FILE_OVAL_OUTPUT:
```

Comment:
Is this for purposes of having an easy switch back to multiple outputs? But still using one template file?

Reply:
Exactly. I have updated the PR description.